Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/CustomMinInputFilter.java

```diff
@@ -0,0 +1,45 @@
+/**
+ *  Paintroid: An image manipulation application for Android.
+ *  Copyright (C) 2010-2015 The Catrobat Team
+ *  (<http://developer.catrobat.org/credits>)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.paintroid.ui.tools;
+
+import android.text.InputFilter;
+import android.text.Spanned;
+import android.util.Log;
+
+public class CustomMinInputFilter implements InputFilter {
```

Comment:
Seems like this class could be reused or merged with `NumberRangeFilter`.
We currently have a max size for images, this should be respected by the EditTexts as well.